python/openmldb/driver.py

```diff
@@ -59,6 +59,16 @@ def dropDB(self, db):
             return True, "ok"
         else:
             return False, status.msg
+        
+    def getAllTables(self):
+        if not self.sdk:
+            raise Exception("please init driver first")
+        status = sql_router_sdk.Status()
+        all_tables = self.sdk.GetAllTables(status)
+        if not all_tables:
```

Comment:
Check status instead of the return value of `all_tables`.

Reply:
updated.